modules/ROOT/pages/dataweave-formats-excel.adoc

```diff
@@ -186,52 +189,75 @@ The following image shows the Excel table output.
 image::dataweave-formats-exceltable.png[]
 
 
-// CONFIG PROPS ///////////////////////////////////////////////////////
+== Limitations
+
+ * Macros are currently not supported.
+ * Charts are ignored.
+ * Pivot tables are not supported.
+
+ // CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
+
 
 [[properties]]
 == Configuration Properties
 
-DataWeave supports the following configuration properties for Excel.
+DataWeave supports the following configuration properties for this format.
+
 
-=== Reader Properties (for Excel)
+[[reader_properties]]
+=== Reader Properties
+
+This format accepts properties that provide instructions for reading input data.
 
-The Excel format accepts properties that provide instructions for reading input data.
 
 [cols="1,1,1,3a", options="header"]
 |===
-| Parameter | Type | Default | Description
-| `header` | `Boolean` | `true` | Indicates whether the Excel table contains
-headers. Valid values are `true` or `false`.
-| `ignoreEmptyLine` | `Boolean` | `true` | Indicates whether to ignore empty
-line. Valid values are `true` or `false`.
-| `streaming` | `Boolean` | `false` | Introduced in Mule 4.2.2: Streaming is intended for processing a large file. When streaming is enabled, the reader accesses each row sequentially, keeping one row in memory at a time, instead of making all data available at once. Streaming does not permit random access to rows in the file. Use only if the entries are accessed sequentially. Valid values are `true` or `false`.
-| `tableOffset` | `String` | `null` | The position of the first cell in the
-table (`<Column><Row> example A1`).
-| `zipBombCheck` | `Boolean` | `true` | If set to `false`, the zip bomb check is turned off. Valid values are `true` or `false`.
+|Parameter |Type |Default|Description
+|`header`  |`Boolean`|`true`|Indicates whether the first line of the output contains header field names.
+
+Valid values are `true` or `false`.
+|`ignoreEmptyLine`  |`Boolean`|`true`|Indicates whether any empty line should be ignored.
+
+Valid values are `true` or `false`.
+|`streaming`  |`Boolean`|`false`|Streams input when set to `true`. Use only if entries are accessed sequentially. The input must be a top-level array. For more on streaming in DataWeave, see https://docs.mulesoft.com/mule-runtime/4.3/dataweave-formats#dw_readers_writers[DataWeave Readers].
+
+Valid values are `true` or `false`.
+|`tableLimit`  |`String`|`'Unbounded'`|Position of the last column in each row. Accepts a pattern `<Column>` (for example, 'A' or 'AB'), a 'HeaderSize' that uses the location of the last header,or 'Unbounded' each row will consume 
```

Comment:
TODO: tableLimit  How to set these or what "'Unbounded' each row will consume" means

@mlischetti